playpen/ansible/roles/qpidd/tasks/main.yml

```diff
@@ -2,10 +2,15 @@
 - name: Install packages
   dnf: name={{ item }} state=present
   with_items:
-    - qpid-cpp-server
-    - qpid-cpp-server-store
+    # qpid is broken in f23, so we have to install the compat packages:
+    #  https://bugzilla.redhat.com/show_bug.cgi?id=1286881
+    # - qpid-cpp-server
+    # - qpid-cpp-server-linerstore
+    - compat-qpid-cpp-server
+    - compat-qpid-cpp-server-store
```

Comment:
The F23 box is so significantly different from purpleidea's F22 box that I opted to give up on backward compatibility between boxes. My assumption is that now that we're tracking the official fedora cloud images we've got a good excuse to start fresh at this point.